libgalois/include/katana/Galois.h

```diff
@@ -24,5 +24,6 @@
 #include "katana/Mem.h"
 #include "katana/SharedMemSys.h"
 #include "katana/config.h"
+#include "katana/Version.h"
```

Comment:
Let's not add this to the main include. See my comment below for how to handle it instead.